host_applications/linux/apps/gencmd/gencmd.c

```diff
@@ -36,41 +35,59 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "interface/vmcs_host/vc_vchi_gencmd.h"
 
+void show_usage()
+{
+   printf( "Usage: vcgencmd [-t] command\n" );
+   printf( "Send a command to the VideoCore and print the result.\n\n" );
+   printf( "  -t          Time how long the command takes to complete\n" );
+   printf( "  -h, --help  Show this information" );
```

Comment:
You've missed the newline, but these could all be puts() calls which wouldn't need the newline.